### PR TITLE
(DOCSP-15200) Realm Sync Monitoring Endpoints

### PR DESCRIPTION
**Pull request by nlarew**
## Pull Request Info

- The primary workflow for this ticket is covered by the monitoring/alerts project, but adding these endpoints for completeness

### Jira

- (DOCSP-15200) Realm Sync Monitoring Endpoints

### Staged Changes (Requires MongoDB Corp SSO)

- [OpenAPI > Event Processors](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/sync-monitoring-endpoints/admin/api/v3/#tag/event-subscriptions)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/openapi-admin-v3.yaml [hunk 1]
```diff
@@ -5773,6 +5910,9 @@ tags:
   - name: endpoints
     x-displayName: Endpoints
     description: View and manage your app's custom [HTTPS endpoints](https://www.mongodb.com/docs/atlas/app-services/data-api/custom-endpoints/).
+  - name: event-subscriptions
+    x-displayName: Event Subscriptions
+    description: View your app's active event subscriptions.
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:5915]: can you add another sentence or two explaining what an event subscription is?

File: source/openapi-admin-v3.yaml [hunk 2]
```diff
@@ -351,6 +351,104 @@ paths:
...
+          The type of event subscription to return.
+        in: query
+        required: false
+        schema:
+          type: string
+          enum:
+            - DATABASE
+            - AUTHENTICATION
+            - SCHEDULED
+            - SYNCTRANSLATOR
```
[the remark this answers is not in the document]
Comment (nlarew) [reply source/openapi-admin-v3.yaml:370]: There's no standard way to do this actually, but I think we can use Redoc's `x-enumDescriptions` extension to do this!

[the remark this answers is not in the document]
Comment (mongodben) [reply source/openapi-admin-v3.yaml:370]: i meant like in this field

```yaml
description: |-
          The type of event subscription to return. ADD OTHER DESCRIPTION HERE
```

but your way cool too

File: source/openapi-admin-v3.yaml [hunk 3]
```diff
@@ -351,6 +351,109 @@ paths:
...
+          x-enumDescriptions:
+            DATABASE: Return database triggers
+            AUTHENTICATION: Return authentication triggers
+            SCHEDULED: Return scheduled triggers
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:374]:
```suggestion
            DATABASE: Return Database Triggers
            AUTHENTICATION: Return Authentication Triggers
            SCHEDULED: Return Scheduled Triggers
```

product names Title Case!

File: source/openapi-admin-v3.yaml [hunk 4]
```diff
@@ -5773,6 +5915,12 @@ tags:
   - name: endpoints
     x-displayName: Endpoints
     description: View and manage your app's custom [HTTPS endpoints](https://www.mongodb.com/docs/atlas/app-services/data-api/custom-endpoints/).
+  - name: event-subscriptions
+    x-displayName: Event Subscriptions
+    description: |-
+      View your app's active event subscriptions. These monitor
+      real-time events, e.g. from a MongoDB change stream, to power
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:5922]:
```suggestion
      real-time events, like from a MongoDB change stream, to power
```

**Review by mongodben — APPROVED** [review]
lgtm. some small changes to make. but nothing blocking

**Pull request merged**